lib/src/models/crypto.ts

```diff
@@ -27,3 +29,19 @@ export interface JWKInterface {
     alg: string;
     n: string;
 }
+
+export interface CryptoUtils {
+    getCodeVerifier(): string;
+    getCodeChallenge(verifier: string): string;
+    getJWKForTheIdToken(jwtHeader: string, keys: JWKInterface[]): Promise<any>;
+    isValidIdToken(
+        idToken: string,
+        jwk: any,
+        clientID: string,
+        issuer: string,
+        username: string,
+        clockTolerance: number | undefined,
+        supportedAlgorithms: string[]
```

Comment:
Would it be better to remove `supportedAlgorithms` from the function signature and import in the respective environment specific SDK?

Reply:
Yes, that makes sense.

Reply:
fixed